verl/workers/sharding_manager/megatron_vllm.py

```diff
@@ -172,7 +173,14 @@ def __enter__(self):
             from verl.utils.vllm.patch import patch_vllm_moe_model_weight_loader
 
             patch_vllm_moe_model_weight_loader(model)
-            loaded_params = model.load_weights(per_tensor_param)
+
+            if is_fp8_model(self.model_runner.vllm_config):
```

Comment:
The sharding manager has been deprecated in https://github.com/volcengine/verl/pull/3285, please do no modify it anymore.